scripts/libs/tests/testingData/existing-changelog.md

```diff
@@ -0,0 +1,2650 @@
+# Changelog for the Mapbox Navigation SDK for Android
```

Comment:
It should be sufficient to keep just a part of the file, tests never interact with the end of this file anyway